src/control-plane/backend/lambda/api/model/metadata.ts

```diff
@@ -77,14 +79,18 @@ export interface IMetadataUserAttribute {
   readonly projectId: string;
   readonly appId: string;
 
+  readonly eventName: string;
+
   readonly attributeId: string;
   readonly name: string;
   displayName: string;
   description: string;
   readonly metadataSource: MetadataSource;
   hasData: boolean;
   readonly valueType: MetadataValueType;
-  valueEnum: IMetadataAttributeValue[];
+  valueEnum?: string[];
```

Comment:
Is it still useful to keep it?

Reply:
we need read it from DDB and then patch display info to values.